inc/class-wpseo-utils.php

```diff
@@ -933,23 +933,36 @@ public static function get_icon_svg( $base64 = true ) {
 		return $svg;
 	}
 
+	/**
+	 * Checks if the WP-REST-API is available.
+	 *
+	 * @since 3.6
```

Comment:
not sure why two since params, but not regression. was the result of a copy-paste.